token/LunchToken.sol

```diff
@@ -412,9 +510,17 @@ contract LunchToken is Context, IERC20, IERC20Metadata {
         return (false, 0);
     }
 
-    function getParticipantName(address address_) public view onlyOwner returns (string memory) {
+    function getParticipantName(address address_)
+        public
+        view
+        onlyOwner
+        returns (string memory)
+    {
         (bool _isParticipant, uint256 i) = isParticipant(address_);
-        return _isParticipant ? participantNames[i] : "This address is not participating in the voting.";
+        return
+            _isParticipant
+                ? participantNames[i]
+                : "This address is not participating in the voting.";

```

Comment:
making this message shorter would also affect gas usage?

Reply:
answer from @DevABDee:

(As of now, in Solidity, `Custom Errors` don't work with Ternary Operations. I mean Ternary Satements cannot revert custom errors.)
Yeah you should definitely shorten the string/err msg or maybe use if-else with custom err to save some more gas.